tests/test_model.py

```diff
@@ -47,10 +47,15 @@ def test_sd2_generate(guidance_scale, negative_prompt):
     assert output.shape == (1, 3, 8, 8)
 
 
-def test_sdxl_forward():
+@pytest.mark.parametrize('mask_pad_tokens', [True, False])
```

Comment:
Should we also test this with the SD2 forward and generate?

Reply:
I need to redo the tests so we don't build so many models first. A few more and the tests will start timing out again haha

Reply:
SG!